aladdin.sh

```diff
@@ -88,6 +65,25 @@ function check_cluster_alias() {
     fi
 }
 
+function get_k3d_ports() {
```

Comment:
Configurable ports for k3d startup. Did not want to default to using port 80 since that may be in use on the host already, but this allows people the option of overriding to that, which allows them to access services locally using port 80